### PR TITLE
Updates to origin trials articles

### PR DESCRIPTION
**Pull request by samdutton**

### CODE REVIEW

File: site/en/blog/origin-trials/index.md [hunk 1]
```diff
@@ -56,16 +56,18 @@ available in Chrome by default. Sites needing extra time to migrate off AppCache
...
 
-## How to register for an origin trial
+## Take part in an origin trial
 
 1. Choose an origin trial from the [list of active trials](https://developers.chrome.com/origintrials/#/trials/active).
-1. Request a token by clicking the **Register** button and filling out the form.
-1. Add the token to your web pages,
-   using one of the following methods:
-   -  As a meta tag in the &lt;head&gt; of each page served:
+1. Request a token (a string of letters and numbers) by clicking the **Register** button and
```
Comment (heyawhite) [inline site/en/blog/origin-trials/index.md:62]: Does this definition matter to this audience? I'd guess the developers who will run these instructions know what to expect?

File: site/en/blog/origin-trials/index.md [hunk 2]
```diff
@@ -56,16 +56,18 @@ available in Chrome by default. Sites needing extra time to migrate off AppCache
...
+filling out the form.
+1. Provide the token on every web page for which you want the trial feature to be enabled, using one
```
Comment (heyawhite) [inline site/en/blog/origin-trials/index.md:64]:
```suggestion
1. Provide the token to every web page participating in this trial, using one
```

Comment (heyawhite) [reply site/en/blog/origin-trials/index.md:64]: The "who" for indicating participation is clear at the start, so it doesn't need to be stated. This modification slightly shortens and clears up your intent.

File: site/en/blog/origin-trials/index.md [hunk 3]
```diff
@@ -75,24 +77,51 @@ the [deprecation trial to continue using AppCache](/origintrials/#/view_trial/17
...
-The final shipping API might be worse for it.
-
-In rare circumstances, if there was clear evidence that developers engaged with the origin trial and that their
-concerns were taken into account in the final API design and implementation,
-this breakage period may be skipped
-[upon request](https://sites.google.com/a/chromium.org/dev/blink/launching-features#sites-canvas-main-content:~:text=If%20you%20wish%20to%20skip%20the,Ship%20imply%20approval%20of%20the%20request.).
+
+Usually if an API lands unchanged after a successful origin trial, there is a
+short period between the end of the origin trial and the date the implementation ships in the
+browser when the API will not be available. This is by design. If Chrome were to avoid the
```
Comment (heyawhite) [inline site/en/blog/origin-trials/index.md:98]:
```suggestion
browser when the API is unavailable. This is by design. If Chrome were to avoid the
```

File: site/en/blog/third-party-origin-trials/index.md [hunk 4]
```diff
@@ -73,18 +76,41 @@ on progress with third-party origin trials.
...
+
+
+## Provide feedback
 
 If you're registering for a third-party origin trial and have feedback to share on the process or
-ideas on how we can improve it, [create an
-issue](https://github.com/GoogleChrome/OriginTrials/issues/new) on the Origin Trials GitHub code
-repo.
+ideas on how we can improve it, [create an issue](https://github.com/GoogleChrome/OriginTrials/issues/new)
+on the Origin Trials GitHub code repo.
```
Comment (heyawhite) [inline site/en/blog/third-party-origin-trials/index.md:106]:
```suggestion
ideas on how we can improve it, [create an Issue](https://github.com/GoogleChrome/OriginTrials/issues/new)
on the Origin Trials GitHub repository.
```

File: site/en/blog/third-party-origin-trials/index.md [hunk 5]
```diff
@@ -73,18 +76,41 @@ on progress with third-party origin trials.
...
 ## Find out more
 
+-  [Demo](https://ot-3p.glitch.me)
 -  [Getting started with Chrome's origin trials](/blog/origin-trials/)
 -  [Origin Trials Guide for Web Developers](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md)
 -  [Chrome Platform Status](https://www.chromestatus.com/features/5691464711405568)
 
-Photo by [Louis Reed
-](https://unsplash.com/@_louisreed) on [Unsplash](https://unsplash.com/photos/JeInkKlI2Po).
+Photo by [Louis Reed](https://unsplash.com/@_louisreed) on [Unsplash](https://unsplash.com/photos/JeInkKlI2Po).
```
[the remark this answers is not in the document]
Comment (heyawhite) [reply site/en/blog/third-party-origin-trials/index.md:116]: This matches your change in the other blog post.

**Review by heyawhite — APPROVED** [review]
Small suggestions for clarity / style. Otherwise LGTM.

File: site/en/blog/third-party-origin-trials/index.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (samdutton) [reply site/en/blog/third-party-origin-trials/index.md:116]: This change was just because the picture no longer seems to be displayed on unsplash.com, not sure why.

File: site/en/blog/origin-trials/index.md [hunk 6]
```diff
@@ -75,24 +77,51 @@ the [deprecation trial to continue using AppCache](/origintrials/#/view_trial/17
...
+described earlier, you can inject a token with JavaScript:
+
+```javascript
+const otMeta = document.createElement('meta');
+otMeta.httpEquiv = 'origin-trial';
+otMeta.content = 'TOKEN_GOES_HERE';
+document.head.append(otMeta);
+```
+
+This technique is used to provide tokens for [third-party trials](#third-party).
```
[the remark this answers is not in the document]
Comment (samdutton) [reply site/en/blog/origin-trials/index.md:91]: It's tricky. The most common reason for providing a token via JavaScript  (not in a header or in HTML) is for third-party tokens, but I realised that sites may want to provide tokens via JavaScript for other reasons. I wanted to make this a bit clearer: you will need to use JavaScript to inject a token programmatically if you're a third party participating in an origin trial on multiple sites,  but you can also use JavaScript to inject tokens on your own sites if you prefer.

File: site/en/blog/origin-trials/index.md (hunk 2 above)
Comment (samdutton) [reply site/en/blog/origin-trials/index.md]: Agree that this has become a bit wordy, but I'm uncomfortable with 'every web page participating'. (It's 'you', the developer, who is participating). Also — nit — the token is not provided 'to' the web page: it's made available on the web page 'to' the browser to enable the feature.

I've changed it to 'Provide the token on every web page for which you want the trial feature to be enabled: ...'

File: site/en/blog/origin-trials/index.md (hunk 1 above)
Comment (samdutton) [reply site/en/blog/origin-trials/index.md]: You're right — I deleted ' (a string of letters and numbers)'.

File: site/en/blog/origin-trials/index.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (samdutton) [reply site/en/blog/origin-trials/index.md]: I've left the format as-is but updated the text so the language is a bit more direct/active, to match your wording.
